app/src/organisms/UpdateRobotSoftware/UpdateSoftware.tsx

```diff
@@ -52,6 +48,13 @@ export function UpdateSoftware({
       height="33rem"
       borderRadius={BORDERS.borderRadius12}
     >
+      <Icon
+        name="ot-spinner"
+        size="5rem"
+        spin={true}
```

Comment:
Basically, we just pass `spin` to `Icon`

```suggestion
        spin
```